src/PayPal/Provider.php

```diff
@@ -67,7 +67,7 @@ protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
             'id'       => str_replace('https://www.paypal.com/webapps/auth/identity/user/', null, $user['user_id']),
-            'nickname' => null, 'name' => $user['name'],
+            'nickname' => null, 'name' => $user['name'] ?? '',
```

Comment:
can we split nickname and name on their own lines?
also, I think null as the default makes more sense.

Reply:
Sure thing